easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCC-4.8.2.eb

```diff
@@ -6,6 +6,9 @@ description = """The Ncurses (new curses) library is a free software emulation o
  and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
  function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
 
+# in relation to -no-cpp-precomp bug: https://lists.gnu.org/archive/html/bug-ncurses/2012-12/msg00001.html
+patches = ['ncurses-5.9_configure_darwin.patch']
```

Comment:
@wpoely86: you may want to integrate this bit in #1337 too, while you're at it... :-)